src/js/components/LoginPage/Login.scss

```diff
@@ -0,0 +1,127 @@
+$primary-color: #fff;
+
+.login{
+  background-color: #bbb;
+  min-height: 100vh;
+  color: $primary-color;
+  padding-bottom: 30px;
+  & span {
+    display: block;
+    overflow: hidden;
+    padding: 0 4px 0 6px
+  }
+}
+
+label {
+  float: left
+}
+
+input {
+  width: 100%
+}
```

Comment:
add empty line between declarations

Reply:
Thank you I have put the new lines between the declaration, please give me feedback or if possible resolve this conversation.